pkg/game/board_test.go

```diff
@@ -147,3 +147,59 @@ func TestBoardPlaceTilePlacesTwoTilesOfSameTypeProperly(t *testing.T) {
 		t.Fatalf("expected %#v, got %#v instead", expected, actual)
 	}
 }
+
+/*
+*
+Test creating simple loop with starting tile
+using straight roads and road turns
+*/
+func TestBoardScoreRoad(t *testing.T) {
```

Comment:
I'd suggest adding a few more tests:
- test using a + shaped tile (four roads in different directions that are not part of the same feature)
- test where more than one player has a maximum number of meeples on the road
- test with an incomplete road
- test with a complete road that is not a loop
- maybe a test where a road doesn't have any meeples? (Not sure is such a situation may even happen)

It doesn't need to be 4 different tests, as some of these situations may be merged into one test.

Reply:
addedd more tests

Reply:
Please also add (or modify) a test when there are meeples belonging to different players on the same road and at least two of the players have a maximum number of meeples (so two players are awarded points for this road)

Reply:
first test modified

Reply:
Still only one player is being awarded points for the road

Reply:
in scoring loop, both players are tested